Formula/y/yo.rb

```diff
@@ -0,0 +1,23 @@
+require "language/node"
+
+class Yo < Formula
```

Comment:
@chenrui333  concerning the failed test, is deprecated  https://formulae.brew.sh/cask/yo

```shell
Full audit yo --online --new output
  Error: No such file or directory @ rb_check_realpath_internal - /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks/yo.rb
```

how to handle this correctly? should the whole formula be renamed to `yeoman`?

Reply:
you can add `  conflicts_with cask: "yo"`